lib/Command/ResetFaces.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace OCA\Recognize\Command;
+
+use OCA\Recognize\Db\FaceDetectionMapper;
+use OCA\Recognize\Service\TagManager;
+use OCP\IDBConnection;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ResetFaces extends Command {
+	private IDBConnection $db;
+
+	public function __construct(IDBConnection $db) {
+		parent::__construct();
+		$this->db = $db;
+	}
+
+	/**
+	 * Configure the command
+	 *
+	 * @return void
+	 */
+	protected function configure() {
+		$this->setName('recognize:reset-faces')
+			->setDescription('Remove all face detections from previously classified files');
+	}
+
+	/**
+	 * Execute the command
+	 *
+	 * @param InputInterface  $input
+	 * @param OutputInterface $output
+	 *
+	 * @return int
+	 */
+	protected function execute(InputInterface $input, OutputInterface $output): int {
+		try {
+			$qb = $this->db->getQueryBuilder();
+			$qb->delete('recognize_face_detections')
+				->executeStatement();
+			$qb = $this->db->getQueryBuilder();
+			$qb->delete('recognize_face_clusters')
+				->executeStatement();
```

Comment:
I would say DB request in the command is a no-go.
Call the controller if possible, or move the code to a service that gets called from both controller and command?

Reply:
Ah, yeah, that part is a bit dirty. Thanks for catching that :)

Reply:
Generally, thanks for your diligent reviews! :hearts: I know it's a lot at the moment but I'm really grateful for your pair of eyes :pray: